project.mml

```diff
@@ -1184,7 +1185,8 @@ Layer:
             AND boundary = 'administrative'
             AND admin_level = '2'
             AND name IS NOT NULL
-            AND way_area > 100*!pixel_width!::real*!pixel_height!::real
+            AND way_area > 100*POW(!scale_denominator!*0.001*0.28,2)
```

Comment:
These changes and those following are fine. Using scale_denominator for country_names, state_names, text etc makes sense, so that the display of these features will be consistent at different resolutions.